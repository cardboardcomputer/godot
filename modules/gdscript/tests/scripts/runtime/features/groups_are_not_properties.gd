# https://github.com/godotengine/godot/issues/73843
extends RefCounted

@export_group("Resource")
@export_category("RefCounted")

func test():
	prints("Not shadowed", Resource.new())
	prints("Not shadowed", RefCounted.new())
