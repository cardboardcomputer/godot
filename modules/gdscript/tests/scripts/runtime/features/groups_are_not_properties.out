GDTEST_OK
Not shadowed <Resource#-9223371975785708326>
Not shadowed <RefCounted#-9223371975768931110>
